data/lua/core/map.lua

```diff
@@ -28,6 +39,10 @@ if wesnoth.kernel_type() ~= "Application Lua Kernel" then
 	-- A        A^       A^B      ^        ^B
 	-- implied mode:
 	-- both     base     both     overlay  overlay
+	
+	---Adjusts a terrain code to produce one that will replace the base terrain only
```

Comment:
I think this should be "that will replace the base terrain only, when the adjusted code is used as an argument to wesnoth.map.some_function".

There seem to be no callers of this function in mainline, so I can't check my assumption against example code.